data_steward/resource_files/schemas/rdr/cdr_etl_run_code_info.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "type": "string",
+    "name": "codeType",
+    "mode": "nullable",
+    "description": ""
```

Comment:
I am not taking this code review, but I would love to see some sort of description of what we think this field is.